src/Management/test/Endpoint.Test/Actuators/Refresh/TestConfigurationSource.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+using System.Globalization;
+using Microsoft.Extensions.Configuration;
+
+namespace Steeltoe.Management.Endpoint.Test.Actuators.Refresh;
+
+/// <summary>
+/// Tracks how many times the provider has been loaded.
+/// </summary>
+internal sealed class TestConfigurationSource : IConfigurationSource
+{
+    private readonly TestConfigurationProvider _provider = new();
+
+    public IConfigurationProvider Build(IConfigurationBuilder builder)
+    {
+        return _provider;
+    }
+
+    private sealed class TestConfigurationProvider : ConfigurationProvider
+    {
+        private int _loadCount;
+
+        public override void Load()
+        {
+            base.Load();
+
+            int newLoadCount = Interlocked.Increment(ref _loadCount);
+            Data["FakeLoadCount"] = newLoadCount.ToString(CultureInfo.InvariantCulture);
```

Comment:
"FakeLoadCount" = "How many times this fake was loaded", right?
What about "ConfigurationLoadCount" or even just "LoadCount" since it is a real count of how many times Load was called, and the fact it's inside a fake is (probably?) not super important

Reply:
It's the Fake provider, tracking how many times its Load method was called. This doesn't apply to every provider. For example, ConfigServer reloads itself periodically, so its count would be higher.

Reply:
I forgot to complete my original thought that it might be useful to use the same thing or kind of thing with Placeholder and/or other wrapping config providers, but there's no need to make additional changes now, this is fine